.drone.yml

```diff
@@ -29,7 +29,7 @@ pipeline:
     image: antora/antora:1.0.1
     pull: true
     commands:
-      - antora generate --cache-dir cache/ --redirect-facility netlify site.yml
+      - antora generate --cache-dir cache/ --redirect-facility static site.yml
```

Comment:
This change should also be reverted, sorry

Reply:
You want netlify put back? Why? We're not using Netlify now.

Reply:
Master branch is using a make target, so just drop this change :)